spring-social-zotero/src/main/java/org/springframework/social/zotero/api/impl/GroupsTemplate.java

```diff
@@ -264,7 +265,8 @@ public void deleteItem(String groupId, String citationKey, Long citationVersion)
         HttpEntity<JsonNode> dataHeader = new HttpEntity<JsonNode>(headers);
 
         try {
-            restTemplate.exchange(buildUri(url, false), HttpMethod.DELETE, dataHeader, String.class);
+            return restTemplate.exchange(buildUri(url, false), HttpMethod.DELETE, dataHeader, String.class)
+                    .getStatusCode().is2xxSuccessful();
```

Comment:
i'm fairly certain that if the server doesn't respond with a 2xx code, an exception is thrown. have you tried what happens in an unsuccessful case?